suite/tests/api/dis-a64.txt

```diff
@@ -2152,6 +2139,66 @@ fd7fffff : ldr    d31, [sp,#32760]        : ldr    +0x7ff8(%sp)[8byte] -> %d31
 1f3789e4 : fnmsub s4, s15, s23, s2                  : fnmsub %s15 %s23 %s2 -> %s4
 1ff789e4 : fnmsub h4, h15, h23, h2                  : fnmsub %h15 %h23 %h2 -> %h4
 
+# Floating-point conversion
+1e624117 : fcvt s23, d8                             : fcvt   %d8 -> %s23
+1e63c1fd : fcvt h29, d15                            : fcvt   %d15 -> %h29
+1ee2431c : fcvt s28, h24                            : fcvt   %h24 -> %s28
+1ee2c002 : fcvt d2, h0                              : fcvt   %h0 -> %d2
+1e240034 : fcvtas w20, s1                           : fcvtas %s1 -> %w20
+9e240067 : fcvtas x7, s3                            : fcvtas %s3 -> %x7
+1e6402c0 : fcvtas w0, d22                           : fcvtas %d22 -> %w0
+9e640015 : fcvtas x21, d0                           : fcvtas %d0 -> %x21
+0e21c827 : fcvtas v7.2s, v1.2s                      : fcvtas %d1 $0x02 -> %d7
+4e21c920 : fcvtas v0.4s, v9.4s                      : fcvtas %q9 $0x02 -> %q0
+4e61cba5 : fcvtas v5.2d, v29.2d                     : fcvtas %q29 $0x03 -> %q5
+5e21cbde : fcvtas s30, s30                          : fcvtas %s30 -> %s30
+5e61c987 : fcvtas d7, d12                           : fcvtas %d12 -> %d7
+1e200115 : fcvtns w21, s8                           : fcvtns %s8 -> %w21
+9e2002ae : fcvtns x14, s21                          : fcvtns %s21 -> %x14
+1e6003a7 : fcvtns w7, d29                           : fcvtns %d29 -> %w7
+9e600229 : fcvtns x9, d17                           : fcvtns %d17 -> %x9
+0e21a925 : fcvtns v5.2s, v9.2s                      : fcvtns %d9 $0x02 -> %d5
+4e21aa61 : fcvtns v1.4s, v19.4s                     : fcvtns %q19 $0x02 -> %q1
+4e61a971 : fcvtns v17.2d, v11.2d                    : fcvtns %q11 $0x03 -> %q17
+5e21a849 : fcvtns s9, s2                            : fcvtns %s2 -> %s9
+5e61a8f1 : fcvtns d17, d7                           : fcvtns %d7 -> %d17
+1e2800f3 : fcvtps w19, s7                           : fcvtps %s7 -> %w19
+9e280085 : fcvtps x5, s4                            : fcvtps %s4 -> %x5
+1e680148 : fcvtps w8, d10                           : fcvtps %d10 -> %w8
+9e680249 : fcvtps x9, d18                           : fcvtps %d18 -> %x9
+0ea1a926 : fcvtps v6.2s, v9.2s                      : fcvtps %d9 $0x02 -> %d6
+4ea1aa84 : fcvtps v4.4s, v20.4s                     : fcvtps %q20 $0x02 -> %q4
+4ee1a80f : fcvtps v15.2d, v0.2d                     : fcvtps %q0 $0x03 -> %q15
+5ea1a89d : fcvtps s29, s4                           : fcvtps %s4 -> %s29
+5ee1aa0c : fcvtps d12, d16                          : fcvtps %d16 -> %d12
+1e290041 : fcvtpu w1, s2                            : fcvtpu %s2 -> %w1
```

Comment:
OK here's the one from above.
```
$ disasm_a64 1e290041
llvm-mc:   1e290041 fcvtpu w1, s2
capstone:  1e290041 fcvtpu w1, s2
bfd:       1e290041 fcvtpu w1, s2
DynamoRIO: 1e290041 xx $0x1e290041 %x1 %x2 %x0 %x9 -> %x1 %x2 %x0 %x9
```

Looking at this dis-a64 test: so this first part is generated from binutils objdump?  So there is some sort of ground-truth test vs an external decoder already -- though I guess there's no automated checking that the objdump operands match the DR ones (need #4382 for that).  Xref #4396.

Reply:
@AssadHashmi can probably answer this better than I can, as I've just joined this department last week. All I can say is I got the first part by compiling a simple program using the instruction with the respective registers with armclang and used armllvn-objdump.